azure-maven-plugin-lib/src/main/java/com/microsoft/azure/maven/utils/AppServiceUtils.java

```diff
@@ -27,8 +27,8 @@
 
 public class AppServiceUtils {
 
-    private static final String SERVICE_PLAN_NOT_FOUND = "Failed to get App Service Plan";
-    private static final String UPDATE_APP_SERVICE_PLAN = "Updating app service plan";
+    private static final String SERVICE_PLAN_NOT_FOUND = "Failed to get App Service Plan.";
```

Comment:
keep the same case for name: 'App Service Plan'

Reply:
FIxed